jobs/service-fabrik-broker/spec

```diff
@@ -29,6 +29,7 @@ provides:
   - enable_swarm_manager
   - feature.ServiceInstanceAutoUpdate
   - http_timeout
+  - deployment_action_timeout
```

Comment:
I would prefer it not to put it as a config. Changing this value is not an option as you would want it to be less that the request time of CF/any consuming platform. Putting it in the config will endanger it that someone changes this value and response to broker APIs times out.

Reply:
Make it constant then?

Reply:
@subhankarc for now we believe 80secs should be good enuf, but since this still under development not sure if this value can remain same across IAAs or we might have to tweak this IAAS specific. I recommend we keep it this way, since we are the owners of landscape config as well changing this is upto us and not left to service owners, just gives us a bit more flexibility. But i certainly agree with your comments that this is sth that should not be played around with.

Reply:
yes. fine.